libraries/shared/storage/fetcher/geth_rpc_storage_fetcher.go

```diff
@@ -83,8 +83,8 @@ func (fetcher GethRpcStorageFetcher) FetchStorageDiffs(out chan<- types.RawDiff,
 				logrus.Trace(fmt.Sprintf("iterating through %d Storage values on account", len(account.Storage)))
 				for _, accountStorage := range account.Storage {
 					diff, formatErr := fetcher.formatDiff(account, stateDiff, accountStorage)
-					logrus.Tracef("adding storage diff to out channel. keccak of address: %v, block height: %v, storage key: %v, storage value: %v",
-						diff.HashedAddress.Hex(), diff.BlockHeight, diff.StorageKey.Hex(), diff.StorageValue.Hex())
+					logrus.Tracef("adding storage diff to out channel. block height: %v, storage key: %v, storage value: %v",
```

Comment:
probably not too important since it's a trace, but would be cool to still include the address in the log if it's available